packages/semantic-tokens/src/control/tokens.ts

```diff
@@ -390,13 +389,13 @@ export const cornerWindowDefault = `var(${cornerWindowDefaultRaw})`;
 export const cornerFlyoutRest = `var(${cornerFlyoutRestRaw})`;
 export const cornerLayerDefault = `var(${cornerLayerDefaultRaw})`;
 export const cornerCardRest = `var(${cornerCardRestRaw})`;
-export const cornerCtrlRest = `var(${cornerCtrlRestRaw}, ${borderRadiusMedium})`;
+export const cornerCtrlRest = `var(${cornerCtrlRestRaw})`;
```

Comment:
Let's retain the original values here and leave them in Button/CompoundButton, we just need to make variants for TextArea, not remove the original fallback